packages/react/src/lib/accessibility/Behaviors/Checkbox/checkboxBehavior.ts

```diff
@@ -3,26 +3,29 @@ import { Accessibility } from '../../types'
 
 type CheckboxBehaviorProps = {
   checked: boolean
+  disabled: boolean
```

Comment:
shouldn't be disabled an optional prop? If it is not passed then we don't render `aria-diabled`